.github/workflows/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Release
+
+on:
+  pull_request:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-latest
+    if: ((github.event_name == 'push' && startsWith(github.ref, 'refs/tags')) || contains(github.event.pull_request.labels.*.name, 'Build wheels'))
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Install python-build and twine
+      run: python -m pip install pip build "twine>=3.3" -U
+
+    - name: Build package
+      run: python -m build --sdist --wheel .
+
+    - name: List result
+      run: ls -l dist
+
+    - name: Check long_description
+      run: python -m twine check --strict dist/*
+
+    # FIXME: Failed due to astropy-helpers doing things.
+    #- name: Test package
+    #  run: |
+    #    cd ..
+    #    python -m venv testenv
+    #    testenv/bin/pip install pytest pytest-astropy mimeparse requests_mock pillow pyvo/dist/*.whl
+    #    testenv/bin/pytest pyvo
+
+    - name: Publish distribution 📦 to PyPI
+      if: startsWith(github.ref, 'refs/tags')
```

Comment:
Shouldn't check here if the tag version matches the versions in `setup.cfg` like it does in #266?

Reply:
Oh, I overlooked the fact that you still uses the old infrastructure without `setuptools_scm`. Ops! Maybe this PR is not ideal then unless #285 can be done first.